src/burn/engine/logging.cpp

```diff
@@ -480,6 +478,23 @@ extern "C" LPCSTR LoggingPackageRegistrationStateToString(
     }
 }
 
+extern "C" LPCSTR LoggingMsiFileVersioningToString(
+    __in BOOTSTRAPPER_MSI_FILE_VERSIONING fileVersioning
+    )
+{
+    switch (fileVersioning)
+    {
+    case BOOTSTRAPPER_MSI_FILE_VERSIONING_MISSING_OR_OLDER:
+        return "o";
```

Comment:
There are constants that might look prettier: https://docs.microsoft.com/en-us/windows/win32/api/msi/nf-msi-msireinstallproductw

Reply:
You want the value, like `4`? Or the string `"REINSTALLMODE_FILEOLDERVERSION"`? I'm not sure that either is better.

Reply:
I said `might`. :) Agreed, it's not really better...